### PR TITLE
Feat/sponsor component#46

### PR DESCRIPTION
**Pull request by jlsanchezsanz**
Sponsor component and positioning styles #46

### CODE REVIEW

**Comment by jorgecasar** [comment]
This use `iron-ajax` and mocks instead of `polymerday-data`.

Review the commits, please

File: src/polymerday-sponsors-list.html [hunk 1]
```diff
@@ -36,13 +76,31 @@
...
-        sponsorsTitle: String
+
+        sponsorType: String,
+
+        _title: {
+          type: String,
+          computed: '_computedTitle(sponsorType)'
+        },
+
+        _titlesMap: {
```
Comment (jorgecasar) [inline src/polymerday-sponsors-list.html]: This create a dependency, you don't know need to know the sponsor types. It should be a property

File: src/polymerday-sponsors-list.html [hunk 2]
```diff
@@ -1,32 +1,72 @@
...
-  </template>
+    <style>
+
+      .sponsors--container {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: flex-start;
+      }
+
+      .sponsor-platinum {
```
Comment (jorgecasar) [inline src/polymerday-sponsors-list.html]: Platinum, gold, iron and paper could change and this stop working. Use a property.

File: src/polymerday-sponsors-list.html [hunk 3]
```diff
@@ -36,13 +76,31 @@
       is: 'polymerday-sponsors-list',
 
       properties: {
```
Comment (jorgecasar) [inline src/polymerday-sponsors-list.html:72]: I guess we need this properties:
- sponsors (Array)
- header (String)
- logos-size (String): Enumerable with option like XL, L, M, S or something like this.

File: src/polymerday-sponsors-list.html [hunk 4]
```diff
@@ -1,32 +1,72 @@
...
+        }
+
+        .sponsor-paper {
+          width: calc(90% / 2);
+        }
+      }
+
+    </style>
+
+    <polymerday-data
```
Comment (jorgecasar) [inline src/polymerday-sponsors-list.html]: Sorry about that, I guess it's better to take out this and avoid to create a dependency with the data access.

**Review by jorgecasar — CHANGES_REQUESTED** [review]

File: src/polymerday-sponsors-list.html (hunk 2 above)
Comment (jlsanchezsanz) [reply src/polymerday-sponsors-list.html]: done

File: src/polymerday-sponsors-list.html (hunk 4 above)
Comment (jlsanchezsanz) [reply src/polymerday-sponsors-list.html]: done

File: src/polymerday-sponsors-list.html (hunk 3 above)
Comment (jlsanchezsanz) [reply src/polymerday-sponsors-list.html:72]: done

File: src/polymerday-sponsors-list.html (hunk 1 above)
Comment (jlsanchezsanz) [reply src/polymerday-sponsors-list.html]: done

**Review by jorgecasar — APPROVED** [review]

**Pull request merged**